### PR TITLE
Release MongooseIM 6.2.1

### PR DESCRIPTION
**Pull request by jacekwegr**
## Highlights

- Enhanced CETS
- Pools configuration
- Traffic shapers update
- Support for MAM v1.1.0
- More reliable testing
- Various improvements and fixes

### CODE REVIEW

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/213945) / elasticsearch_and_cassandra_mnesia / 7cf71bc6a014f5670d95b2e6420d0e2644c1d33a
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/213945/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/213945/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40f9321b9392f7.2024-04-05_12.51.33/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/213992) / elasticsearch_and_cassandra_mnesia / 7d9accc5b09451c16b051347069c7fe0faf1fc95
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/213992/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/213992/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%407d12af05110d.2024-04-05_13.37.20/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,59 @@
+# [MongooseIM 6.2.1](https://github.com/esl/MongooseIM/releases/tag/6.2.0) - 2024-04-09
+
+## Highlights
+
+- Enhanced CETS
+- Pools configuration
+- Traffic shapers update
+- Support for MAM v1.1.0
+- Added Erlang Doctor
```
Comment (chrzaszcz) [inline CHANGELOG.md:9]: I'd remove this from highlights, because it's a low-level debugging tool.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+- More reliable testing
+- Various improvements and fixes
+
+## All changes
+
+### Added
+- RDBMS backend for `mod_caps` (#4211)
+- Commit hash to GraphQL server status (#4221)
+- Erlang Doctor for diagnostics (#4248)
```
Comment (chrzaszcz) [inline CHANGELOG.md:18]:
```suggestion
- Erlang Doctor debugging tool (#4248)
```

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/214556) / elasticsearch_and_cassandra_mnesia / b64f4638ce1438e28af9ed3896a3e7534655b820
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/214556/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/214556/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%4084dbb3e21634.2024-04-10_09.29.06/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+- Pools:
+  - By host type (#4229)
+  - By host config (#4235)
+  - Names to `mongoose_rdbms` (#4231)
+
+### Changed
+- Improved shapers (#4187, #4203, #4213)
+- Enhanced discovery requests handling (#4194)
+- User-friendly errors for internal database issues (#4192)
```
Comment (chrzaszcz) [inline CHANGELOG.md:27]:
```suggestion
- User-friendly errors for internal databases in the GraphQL API (#4192)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,59 @@
...
+- Increase `idle_timeout` for SSE (#4196)
+- Presence management refactor (#4207)
+- MAM implementation update (#4218, #4225)
```
Comment (chrzaszcz) [inline CHANGELOG.md:30]:
```suggestion
- MAM implementation update to version 1.1.0(#4218, #4225)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,59 @@
...
+- Roster management refactor (#4209)
+
+### Fixed
+- CETS improvements:
+  - Pause on all nodes (#4204)
+  - Node cleanup (#4234, #4250, #4251)
+  - Unnecessary logs removal (#4205)
+- MAM lookup (#4191)
```
Comment (chrzaszcz) [inline CHANGELOG.md:38]:
```suggestion
- MAM lookup error handling (#4191)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,59 @@
...
+- MUC light `id` definition for MariaDB (#4195)
```
Comment (chrzaszcz) [inline CHANGELOG.md:39]:
```suggestion
- MUC Light `id` definition for MariaDB (#4195)
```

File: doc/History.md [hunk 7]
```diff
@@ -1,15 +1,19 @@
...
+## 2023-2024: C2S and CETS
 
 Created an alternative to Mnesia RAM-only tables - CETS.
 It allows to run MongooseIM without Mnesia completely in RDBMS+CETS setup.
 
 Moved the C2S implementation to state machine.
 Added Docker image for arm64.
 
+Enhanced CETS, configurable pools, and traffic updates.
+Added Erlang Doctor for better diagnostics.
```
Comment (chrzaszcz) [inline doc/History.md:12]: I don't think it should be a highlight.

File: doc/History.md [hunk 8]
```diff
@@ -1,15 +1,19 @@
...
-## 2023: C2S and CETS
+## 2023-2024: C2S and CETS
 
 Created an alternative to Mnesia RAM-only tables - CETS.
 It allows to run MongooseIM without Mnesia completely in RDBMS+CETS setup.
 
 Moved the C2S implementation to state machine.
 Added Docker image for arm64.
 
+Enhanced CETS, configurable pools, and traffic updates.
```
Comment (chrzaszcz) [inline doc/History.md:11]:
```suggestion
Enhanced CETS, configurable pools, and traffic shaping updates.
```

**Review by chrzaszcz — COMMENTED** [review]
Looks good in general. I added some comments.

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/214599) / elasticsearch_and_cassandra_mnesia / 37b42dc8579e4058c9be0127588a94065c82246c
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/214599/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/214599/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40fa98f3e56518.2024-04-10_10.24.40/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,60 @@
+# [MongooseIM 6.2.1](https://github.com/esl/MongooseIM/releases/tag/6.2.0) - 2024-04-10
```
Comment (chrzaszcz) [inline CHANGELOG.md:1]:
```suggestion
# [MongooseIM 6.2.1](https://github.com/esl/MongooseIM/releases/tag/6.2.1) - 2024-04-10
```

**Review by chrzaszcz — COMMENTED** [review]
Looks good

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/214625) / elasticsearch_and_cassandra_mnesia / 4d0a17fa0d2d99e917c0f97abcb7cc5f84260426
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/214625/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/214625/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40be581f09b1d9.2024-04-10_10.46.35/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/214842) / elasticsearch_and_cassandra_mnesia / 62243de799ea3ae4253e781de1b18d533d28c67f
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/214842/elasticsearch_and_cassandra_mnesia.26.1.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/214842/elasticsearch_and_cassandra_mnesia.26.1.2-amd64/big/ct_run.test%40efa76929b6fa.2024-04-11_08.46.33/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/214910) / elasticsearch_and_cassandra_mnesia / e57b82aeaef4b3c147b7cbc8382434a46d3114f0
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/214910/elasticsearch_and_cassandra_mnesia.26.2.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/214910/elasticsearch_and_cassandra_mnesia.26.2.2-amd64/big/ct_run.test%401150c37efa64.2024-04-11_10.02.53/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/215170) / elasticsearch_and_cassandra_mnesia / 5bce194982dcf1edb05d8bf7d31305a82a0164aa
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/215170/elasticsearch_and_cassandra_mnesia.26.2.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/215170/elasticsearch_and_cassandra_mnesia.26.2.2-amd64/big/ct_run.test%40a49511d99004.2024-04-11_14.30.32/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_26](https://circleci.com/gh/esl/MongooseIM/215208) / elasticsearch_and_cassandra_mnesia / f2c2bd1f3f7f7ca84b62837d3a32f3df155029e8
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4254/215208/elasticsearch_and_cassandra_mnesia.26.2.2-amd64)/ [big](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4254/215208/elasticsearch_and_cassandra_mnesia.26.2.2-amd64/big/ct_run.test%409f89a79a25a5.2024-04-12_07.05.40/index.html)
**OK: 437** / Failed: 0 / **User-skipped: 43** / Auto-skipped: 0

**Pull request merged**